src/utils/remainingQuestions.js

```diff
@@ -0,0 +1,19 @@
+import { quizData } from '../data.js';
+
+let remainingQuestions = 10;
```

Comment:
Can you make it dynamic? You can use questions.length from quizdata

Reply:
Done!